apps/api/v2/src/ee/bookings/2024-08-13/bookings.repository.ts

```diff
@@ -169,4 +169,17 @@ export class BookingsRepository_2024_08_13 {
       },
     });
   }
+
+  async getBookingReferences(bookingId: number) {
```

Comment:
I think we need a new `bookings-references.repository.ts` because bookings repository is supposed to access `booking` prisma model but we are accessing `prisma.bookingReference`.